mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1227,6 +1229,12 @@
     <string name="bookmark_deletion_multiple_snackbar_message_3">Deleting selected folders</string>
     <!-- Bookmark undo button for deletion snackbar action -->
     <string name="bookmark_undo_deletion">UNDO</string>
+    <!-- Bookmark snackbar message for deleting a single item. Parameter is the title of the item being deleted -->
+    <string name="bookmark_delete_single_item">Deleted %s</string>
+    <!-- Bookmark snackbar message for deleting multiple items. Parameter is the number of items being deleted -->
+    <string name="bookmark_delete_multiple_items">Deleted %s items</string>
+    <!-- Bookmark folder deletion dialog title. Parameter is the number of items being deleted -->
+    <string name="bookmark_delete_folder_dialog_title">Are you sure you want to delete %s items?</string>
```

Comment:
We're going to need a new string here since we don't handle plurals correctly
CC @ewachowiak for insights

Reply:
For line 1237, I notice that we say "3 selected" elsewhere in the copy and are able to handle this plural correctly. Would it work to say "Are you sure you want to delete 3 selected items?" If not, I think we should say "Are you sure you want to delete multiple items?"

Reply:
If this string is about 3 selected items, we can use that. If it's anything else than 3, we would have to use "multiple"

Reply:
This is a "variable" string, so yes we will need to say "Are you sure you want to delete multiple items?" I will update the Figma.